.test-infra/terraform/google-cloud-platform/utility-cluster/gke.tf

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "gke" {
+  source = "../google-kubernetes-engine"
+  project    = "apache-beam-testing"
+  network    = "default"
+  subnetwork = "default-f91f013bcf8bd369"
+  region     = "us-central1"
+  cluster_name_prefix = "beam-utility"
+  service_account_id = "beam-github-actions@apache-beam-testing.iam.gserviceaccount.com"
+  cluster_name_override = "beam-utility"
```

Comment:
Maybe one can just create a new `tfvars` file storing these values and have the workflow involve provisioning the Kubernetes cluster first, separate from the strimzi workload.